README.md

```diff
@@ -348,7 +348,8 @@ Represents an audio or MIDI clip. Can be used to start/stop clips, and query/mod
 | /live/clip/fire                          | track_id, clip_id                                                   |                                                                                        | Start clip playback                                                                                                                                  |
 | /live/clip/stop                          | track_id, clip_id                                                   |                                                                                        | Stop clip playback                                                                                                                                   |
 | /live/clip/duplicate_loop                | track_id, clip_id                                                   |                                                                                        | Duplicates clip loop                                                                                                                                 |
-| /live/clip/get/notes                     | track_id, clip_id                                                   | track_id, clip_id, pitch, start_time, duration, velocity, mute, [pitch, start_time...] | Query the notes in a given clip.                                                                                                                     |
+| /live/clip/get/notes                     | track_id, clip_id, [from_time, from_pitch, time_span, pitch_span]   | track_id, clip_id, pitch, start_time, duration, velocity, mute, [pitch, start_time...] | Query the notes in a given clip.                                                                                                                     |
+| /live/clip/get/notes_range               | track_id, clip_id, [from_time, from_pitch, time_span, pitch_span]   | track_id, clip_id, min_start_time, max_end_time, min_pitch, max_pitch                  | Query the note range within a given clip. This is helpful in conjunction with /live/clip/get/notes to paginate and handle a large number of notes    |
```

Comment:
@ideoforms This api was added as a temporary fix for https://github.com/ideoforms/AbletonOSC/issues/88, before PR https://github.com/ideoforms/AbletonOSC/pull/89 was made 

If there are too many notes, `/live/clip/get/notes` would fail. To address this, `/live/clip/get/notes_range` was introduced to determine the range of notes, and later `/live/clip/get/notes` could be called with optional parameters [from_time, from_pitch, time_span, pitch_span] for pagination purposes.

After merging PR #89, this API may no longer be required. However, it would still be beneficial to retain it for those who prefer to implement pagination.

Reply:
Thanks, will have another think on this once #89 is merged 👍